classes/class-wc-connect-utils.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Utils' ) ) {
+	class WC_Connect_Utils {
+		/**
+		 * Encodes an object to JSON. Uses wp_json_encode to escape quotes and then double-escapes them for
+		 * saving in the database
```

Comment:
If this function continues to be relevant (with the JSON vs serialization discussion above), can you elaborate on the need for it? Is there a specific function that WP calls upon retrieval of DB values that requires the double escaping?